ctypes.go

```diff
@@ -58,6 +58,11 @@ type wafObject struct {
 	// packed (apart from breaking all tracers of course)
 }
 
+// isNil determines whether this WAF Object is nil or not.
+func (w *wafObject) isNil() bool {
+	return w._type == wafNilType
+}
```

Comment:
I'm torn whether this naming (which sounds more like go), and keeping the `libddwaf` semantics of `isNull` / `wafNullType`.